classes/patterns/class-issues.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Issues class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Issues pattern includes:
+ * Column, Image, Heading, Paragraph, Media & Text.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Issues extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/issues';
+	}
+
+	/**
+	 * Returns the template for one column.
+	 */
+	public static function get_column_template(): string {
+		return '
+			<!-- wp:column {"verticalAlignment":"center","width":"25%","backgroundColor":"white"} -->
+			<div class="wp-block-column is-vertically-aligned-center has-white-background-color has-background" style="flex-basis:25%">
+				<!-- wp:group {"layout":{"wideSize":"80%","contentSize":"100%"}} -->
+				<div class="wp-block-group"><!-- wp:spacer {"height":"16px"} -->
+					<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer -->
+					<!-- wp:media-text {"mediaWidth":20,"mediaSizeSlug":"thumbnail","isStackedOnMobile":false,"verticalAlignment":"center","imageFill":false} -->
+					<div class="wp-block-media-text alignwide is-vertically-aligned-center" style="grid-template-columns:20% auto"><figure class="wp-block-media-text__media"></figure>
+					<div class="wp-block-media-text__content">
+					<!-- wp:paragraph {"align":"left","placeholder":"' . __( 'Enter text', 'planet4-blocks-backend' ) . '","style":{"typography":{"fontSize":"1rem","fontStyle":"normal","fontWeight":"700"}}} -->
+					<p class="has-text-align-left" style="font-size:1rem;font-style:normal;font-weight:700"></p>
+					<!-- /wp:paragraph --></div></div>
+					<!-- /wp:media-text -->
+					<!-- wp:spacer {"height":"16px"} -->
+					<div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer --></div>
+				<!-- /wp:group --></div>
+			<!-- /wp:column -->
+		';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:group {"backgroundColor":"grey-10","className":"alignfull"} -->
+				<div class="wp-block-group alignfull has-grey-10-background-color has-background">
```

Comment:
A separate issue as the previous line, but putting it here to start a new thread.

We should use
```
"align":"full",
```
in the attributes instead of only putting the `.alignfull` class on it. It uses the editor's alignment control for that.
![Screenshot from 2022-04-20 13-46-21](https://user-images.githubusercontent.com/7604138/164223650-b1bc6bcd-6e32-4c7e-aced-e232d92b097d.png)